### PR TITLE
Contact page

### PR DESCRIPTION
**Pull request by kisphp**

### CODE REVIEW

**Comment by kisphp** [comment]
I tried to pass action url from angular controller. for what ever reason, doesn't work. I get an error about interpolation/concatanation. right now I hate angular

File: app/templates/homepage.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
           <nav class="section--hero__nav">
             <a href="#about">About</a>
             <a href="#login">Login</a>
+            <a href="#contact">Contact</a>
```
Comment (josephrexme) [inline app/templates/homepage.html:18]: Do we really need a separate contact page? It should be something we can add within the home page

Comment (kisphp) [inline app/templates/homepage.html:18]: I would keep homepage to tell about the application, about the purpose of the website.

Comment (josephrexme) [inline app/templates/homepage.html:18]: With that there's still not enough content on the homepage to have multiple pages

**Pull request merged**